packages/adapter-netlify/files/entry.js

```diff
@@ -39,3 +42,21 @@ export async function handler(event) {
 		body: 'Not found'
 	};
 }
+
+/**
+ * Separates headers that hold arrays from the others, and removes them from the original headers
+ *
+ * @param {Record<string, string | string[]>} headers
+ * @param  {...string} fields
+ * @returns {Record<string, string[]>}
+ */
+function getMultivalueHeaders(headers, ...fields) {
```

Comment:
I wouldn't expect a function called `getMultivalueHeaders` to modify the input. Can we write this without the side-effect? E.g. if could return an object containing the new `headers` and `multiValueHeaders`

Also, I'd expect the `V` in this function to be capitalized to match the field in the object we're returning

Reply:
Should I leave the fields parameter where it is?